pandas/tests/series/test_timezones.py

```diff
@@ -348,3 +348,19 @@ def test_series_truncate_datetimeindex_tz(self):
         result = s.truncate(datetime(2005, 4, 2), datetime(2005, 4, 4))
         expected = Series([1, 2, 3], index=idx[1:4])
         tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize('copy', [True, False])
+    @pytest.mark.parametrize('method, tz', [
+        ['tz_localize', None],
+        ['tz_convert', 'Europe/Berlin']
+    ])
+    def test_tz_localize_convert_copy_inplace_mutate(self, copy, method, tz):
+        # GH 6326
+        result = Series(np.arange(0, 5),
+                        index=date_range('20131027', periods=5, freq='1H',
+                                         tz=tz))
+        getattr(result, method)('UTC', copy=copy)
```

Comment:
@mroeschke i'm confused by this; wouldn't we expect the tz in expected.index to be UTC, not `tz`

Reply:
The naming of the test is a bit confusing, but this call here should _not_ be an inplace operation.

I'm checking that the original data `result` was not modified after calling `tz_localize/tz_convert` with `copy`

https://github.com/pandas-dev/pandas/issues/6326#issuecomment-34865955

Reply:
ahh i see.  ill update #37498 appropriately